Sources/View/ReusableView.swift

```diff
@@ -17,28 +17,25 @@ extension UITableViewHeaderFooterView: ReusableView {}
 
 public extension UICollectionView {
 
-    func register<T: UICollectionViewCell>(_ cellType: T.Type)
-    where T: ReusableView {
+    func register<T: UICollectionViewCell>(_ cellType: T.Type) {
         register(cellType, forCellWithReuseIdentifier: cellType.reuseIdentifier)
     }
 
     func register<T: UICollectionViewCell>(_ cellType: T.Type)
-    where T: ReusableView, T: NibView {
-        register(cellType.nib, forCellWithReuseIdentifier: cellType.reuseIdentifier)
+        where T: NibView {
```

Comment:
incorrect indentation ( more cases below) 

also, please move this to the same line, or alternatively combine on the  generic parameter, like `<T: UICollectionViewCell & NibView>` (we haven't settled on a chosen variant in our code style yet)